packages/@react-spectrum/combobox/src/MobileComboBox.tsx

```diff
@@ -55,20 +56,39 @@ export const MobileComboBox = React.forwardRef(function MobileComboBox<T extends
     isQuiet,
     isDisabled,
     validationState,
-    isReadOnly
+    isReadOnly,
+    menuTrigger = 'input'
   } = props;
 
   let {contains} = useFilter({sensitivity: 'base'});
   let state = useComboBoxState({
     ...props,
     defaultFilter: contains,
     allowsEmptyCollection: true,
-    shouldCloseOnBlur: false
+    shouldCloseOnBlur: true
   });
 
   let buttonRef = useRef<HTMLElement>();
   let domRef = useFocusableRef(ref, buttonRef);
   let {triggerProps, overlayProps} = useOverlayTrigger({type: 'listbox'}, state, buttonRef);
+  let layout = useListBoxLayout(state);
+  let popoverRef = useRef<HTMLDivElement>();
+  let listBoxRef = useRef<HTMLDivElement>();
+  let inputRef = useRef<HTMLInputElement>();
+
+  let {inputProps: trayInputProps, listBoxProps, labelProps: trayInputLabelProps} = useComboBox(
```

Comment:
I would be very careful about calling this here. `useComboBox` does a bunch of things (e.g. announcements) that might not make sense for the mobile combobox. Plus, manually copying over very specific props is kinda brittle to future changes. Perhaps this would be not needed given my other comment above?

Reply:
I'll see if I can change this, I'll probably need to copy the `onBlur` and `onFocus` logic from useComboBox and replicate it here so that I can fix: `Mobile combobox opens when user clicks the buttons to change the controlled input value` (issue was that we weren't updating the isFocused state when user blurred/focused the button)